defaults-o2-daq.sh

```diff
@@ -42,6 +42,7 @@ overrides:
       - FreeType:(?!osx)
       - Python-modules
       - libxml2
+      - OpenSSL
       - "GCC-Toolchain:(?!osx)"
```

Comment:
libxml + openssl also need to be added to `defaults-o2.sh`

Reply:
Actually, those should come from `Alien-Runtime` in the case of `default-o2.sh`.

Reply:
Yes, but it is usually better to be explicit about dependencies ... and not rely in implicit stuff which is hard to track? In any case, if it already works no problem ...